### ISSUE TITLE
Need to address discordance between shuffling / SHAP results

### ISSUE THREAD

**Issue by kdkorthauer** [mtello22/KorthauerLab#9]
https://github.com/mtello22/KorthauerLab/blob/2167aed8b926a69735fae5c32023b0a0e1ad0890/VMRassociation/SupervisedLearning/Reports/CommitteeReport/CommitteeReport.Rmd#L387

The shuffling results don't exactly align with the SHAP for promoter-based summary models. Why do you think this is? One possible factor here is that the promoter models are already providing close to random / average predictions, so there's not much room for decrease. I'll also remind about the importance of addressing the limitation that perhaps promoter model based on a selection of promoters with highly variable methylation may show increased performance (since the selection of these genes was based on VMRs).

**Comment by mtello22** [comment]
You're right, SHAP analysis may not be very reliable for promoter-based models due to their overall negative R-squared values. I added that argument after the plot of SHAP values.

**Comment by kdkorthauer** [comment]
There are a few points in the intro to Aim 2 and its conclusions where the statements seem to broad/strong (i.e. stating that promoter summary relies heavily on TF, but this isn't supported by Fig 4)
